src/PowerShellEditorServices/Session/PowerShellContext.cs

```diff
@@ -103,6 +103,12 @@ public RunspaceDetails CurrentRunspace
             private set;
         }
 
+        /// <summary>
+        /// Gets the working directory path the PowerShell context was inititially set when the debugger launches.
+        /// This path is used to determine whether a script in the call stack is an "external" script.
+        /// </summary>
+        public string InitialWorkingDirectory { get; private set; }
```

Comment:
This is fine for now, we'll come up with a better way to get the workspace path soon.